File: Data Structures & Algorithms/is-anagram/submission-8.java
class Solution {
    public boolean isAnagram(String s, String t) {
        if (s.length() != t.length()){
            return false;
        }
        int[] seen = new int[26];
        for (int i = 0; i < s.length(); i++){
            seen[s.charAt(i)-'a']++;
            seen[t.charAt(i)-'a']--;
        }

        for (int j : seen){
            if (j != 0){
                return false;
            }
        }
        return true;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-8.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        seen = set(nums)
        res = 0
        for num in nums:
            if (num-1) not in seen:
                streak = 1
                while num + streak in seen:
                    streak += 1
                res = max(res, streak)
        return res